mlflow/genai/judges/instructions_judge/__init__.py

```diff
@@ -79,6 +82,9 @@ def __init__(
             instructions: Natural language instructions for evaluation
             model: The model identifier to use for evaluation (e.g., "openai:/gpt-4")
             description: A description of what the judge evaluates
+            feedback_value_type: Optional type for the 'value' field in the Feedback response.
+                           Default is str. Supported types (FeedbackValueType): int, float,
+                           str, bool, Literal types, dict[str, PbValueType], and list[PbValueType].
```

Comment:
We don't include `PbValueType` in our API docs. Can we say ", as well as dictionaries and lists of these types" or something similar?